.idea/vcs.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
It's good practice to add IDE-specific files to `.gitignore` to exclude them from your repo